internal/api/js/control_messages.go

```diff
@@ -0,0 +1,93 @@
+package js
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+// The unique console log line prefix which denotes a control message to the test rig.
+// The text after this string is a JSON object which contains information on the type
+// of update (e.g a new event, a verification request)
+//
+// For example:
+//
+//	CC:{"t":1,"d":{"RoomID":"!foo:bar","Event":{...}}}
```

Comment:
Do we need the efficiency of using numbered types? I'd prefer `"t": "event"` if there's not a good reason not to.

Reply:
These log lines can be quite long, as they are entire event JSON objects. I'm not sure at what point Chrome will truncate console.log lines, hence trying to keep the header information as small as possible.